datasets/parkeerrechten/parkeerEnVerbijfsRechten/v0.0.1.json

```diff
@@ -0,0 +1,220 @@
+        "required": [
+          "schema",
+          "verworvenRechtId"
+        ],
+        "display": "verworvenRechtId",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "verworvenRechtId": {
+            "type": "integer",
+			"provenance": "verw_recht_id",
+			"description": "Verworven recht Id - Uniek nummer ter identificatie van een parkeer- of verblijfsrecht."
+          },
+          "voertuigId": {
+            "type": "string",
+			"provenance": "vrtg_id_v_recht",
+			"description": "Voertuig Id - Identificatie van het voertuig waarvoor een parkeer- of verblijfsrecht is geregistreerd."
+          },
+          "voertuigRegistratieLandCode": {
+            "type": "string",
+			"provenance": "land_c_v_recht",
+			"description": "Landcode - Identificatie van het land van herkomst van het voertuig waarvoor een parkeer- of verblijfsrecht is geregistreerd. De landcode kan door de rechtverwerver intermediair (provider) aangeleverd worden wanneer een voertuig bijvoorbeeld een buitenlands kenteken heeft."
+          },
+          "gebiedsbeheerderId": {
+            "type": "integer",
+			"provenance": "geb_beh_v_recht",
+			"description": "Gebiedsbeheerdercode - Identificatie van de gebiedsbeheerder waaronder het gebied valt waarvoor een parkeer- of verblijfsrecht is geregistreerd."
+          },
+          "gebiedsbeheerderOmschrijving": {
+            "type": "string",
+			"provenance": "gebieds_beh_oms",
+			"description": "Gebiedsbeheerder - Omschrijving van de gebiedsbeheerder waaronder het gebied valt waarvoor een parkeer- of verblijfsrecht is geregistreerd."
+          },
+          "gebiedCode": {
+            "type": "string",
+			"provenance": "geb_c_v_recht",
+			"description": "Gebiedcode - Identificatie van het gebied waarvoor een parkeer- of verblijfsrecht is geregistreerd."
+          },
+          "gebiedOmschrijving": {
+            "type": "string",
+			"provenance": "gebied_oms",
+			"description": "Gebied - Omschrijving van het gebied waarvoor een parkeer- of verblijfsrecht is geregistreerd."
+          },
+          "gebruiksDoelcode": {
+            "type": "string",
+			"provenance": "gebr_doel_recht",
+			"description": "Gebruiksdoelcode - Identificatie van het gebruiksdoel dat opgegeven is bij de registratie van een parkeer- of verblijfsrecht."
+          },
+          "gebruiksDoelOmschrijving": {
+            "type": "string",
+			"provenance": "gebr_doel_oms",
+			"description": "Gebuiksdoel - Omschrijving van het gebruiksdoel dat opgegeven is bij de registratie van een parkeer- of verblijfsrecht."
+          },
+          "verkooppuntId": {
+            "type": "integer",
+			"provenance": "verk_p_v_recht",
+			"description": "Verkooppunt - Identificatie van het verkooppuntnummer waarmee een parkeer- of verblijfsrecht is geregistreerd."
+          },
+          "lokatie": {
```

Comment:
Moet dit niet `locatie` zijn?

Reply:
Tsja, dat had ik eerder staan, maar de spelling in het voorbeeld van amsterdam schema documentatie https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#geometry noemde het lokatie. Het zijn denk ik beide correcte spellingen, maar ik kies nu de spelling uit het voorbeeld maar. :)

Reply:
Ik ga het voorbeeld aanpassen, want deze spelling mag al sinds 1995 niet meer.

Reply:
Dan ga ik lokatie weer locatie maken :D